### PR TITLE
Split Viam intro page into two sections

### PR DESCRIPTION
**Pull request by npentrel**
https://docs-test.viam.dev/3211/get-started/

### CODE REVIEW

File: docs/get-started/_index.md [hunk 1]
```diff
@@ -112,14 +99,40 @@ Viam uses WebRTC and gRPC for secure peer-to-peer communications across network
...
+
+The `viam-server` executable runs on **any** computer that runs on 64-bit Linux OS, macOS Computer, or Windows Subsystem for Linux.
+Here are some of the boards Viam supports:
+
+{{< board-carousel >}}
+<br>
+
+Viam also supports microcontrollers with the [micro-RDK](/installation/#install-micro-rdk).
+
+## Viam Cloud
```
Comment (sguequierre) [inline docs/get-started/_index.md:112]:
```suggestion
## Viam Cloud App
```

File: docs/get-started/_index.md [hunk 2]
```diff
@@ -112,14 +99,40 @@ Viam uses WebRTC and gRPC for secure peer-to-peer communications across network
...
+### Configuration
+
+As you configure your machine, you can test each resource to confirm it is working as expected in the same UI you use for configuration.
+
+{{<gif webm_src="/test.webm" mp4_src="/test.mp4" alt="Test a camera stream" max-width="600px" class="aligncenter">}}
+
+### Extensibility
+
+For everything that is not built-in, you can deploy _{{< glossary_tooltip term_id="module" text="modules" >}}_ from the [**Viam Registry**](/registry/).
+Modules provide custom components or services as _modular resources_ which are run and managed by `viam-server` as processes.
```
Comment (sguequierre) [inline docs/get-started/_index.md:125]:
```suggestion
Modules provide custom components or services as _modular resources_ which are run and managed by `viam-server` as processes.
```
should we link out to processes here or is that confusing?

**Review by sguequierre — APPROVED** [review]
Looks good

File: docs/get-started/_index.md (hunk 1 above)
Comment (sguequierre) [reply docs/get-started/_index.md:112]: or something like that

File: docs/get-started/_index.md (hunk 2 above)
Comment (npentrel) [reply docs/get-started/_index.md:125]: not that kind of processes, they're run as system processes. Not the processes you configure in the configuration of a machine

File: docs/get-started/_index.md (hunk 1 above)
Comment (npentrel) [reply docs/get-started/_index.md:112]: No, this refers to the Cloud part of the platform

**Pull request merged**